filebeat/module/elasticsearch/server/_meta/fields.yml

```diff
@@ -2,10 +2,12 @@
   description: "Server log file"
   type: group
   fields:
-  - name: component
-    description: "Log component"
-    example: "o.e.c.m.MetaDataCreateIndexService"
+  - name: stacktrace
```

Comment:
I wonder if we should even index the stacktrace? Or only have it as part of the source. Or alternative is this one of the fields that actually should be `text` as I expect it to be pretty long?

Reply:
Perhaps this could be a boolean option for this fileset? Something like `include_stacktrace`?

Reply:
That is an other option in case space becomes an issue. At the same time if there is an error, first thing I do is look at the stacktrace so I think it's good to have it there. If someone does not want it, he could still use a processor to drop the field.

So I rather have it in here, I'm more asking if text is the correct type.

Reply:
Gotcha, I think for now we should not index it. It will still be shown as part of the source. The user would be be able to use other fields to narrow down their search (e.g. `log.level:ERROR' and then view the stack trace.

Reply:
In other words, I don't think it will be too common to want to search for something in the stack trace itself.